src/GetMatch.elm

```diff
@@ -27,55 +23,43 @@ matches =
     ]
 
 
-checkMatch : Maybe Int -> Maybe Match
-checkMatch match =
-    andThen (\n -> getAt (n - 1) matches) match
-
-
-unwrapToString : Maybe Int -> String
-unwrapToString maybeInt =
-    unwrap "" String.fromInt maybeInt
-
-
-maybeIntToInt : Maybe Int -> Int
-maybeIntToInt mi =
-    case mi of
-        Nothing ->
-            0
-
-        Just n ->
-            n
-
+type AllianceColor
+    = Blue
+    | Red
 
-getMatch : Maybe Int -> String -> String
-getMatch match station =
-    stationIndex match <| station
 
+type StationNumber
+    = One
+    | Two
+    | Three
 
-stationIndex : Maybe Int -> String -> String
-stationIndex match station =
-    case checkMatch match of
-        Nothing ->
-            "Not a match"
 
-        Just matchData ->
-            if station == "Blue 1" then
-                String.fromInt matchData.blue.one
+type alias AllianceStation =
+    ( AllianceColor, StationNumber )
```

Comment:
I think `AllianceStation` is kind of misleading. Because, the `AllianceStation` is the whole structure consisting of the 3 alliance members.
Maybe call it `TeamStation` or something more specific for a single team.